### ISSUE TITLE
Allow cloning repository from Gitlab

### ISSUE THREAD

**Issue by pvojnisek** [notebooks-ai/notebooks-help#20]
git clone <url>
Asks for password. Gitlab needs to register a public key. How can I solve this issue?

**Comment by martinzugnoni** [comment]
Thanks @pvojnisek. Gitlab is currently not supported in notebooks.ai, but we will try to implement it soon.

**Comment by santiagobasulto** [comment]
@pvojnisek did you try creating a project out of a gitlab repo, or you used the command line to do a regular `git clone [repo url]`?

**Comment by pvojnisek** [comment]
Thank you for your effort!
@santiagobasulto yes, exactly. SSH public key has to be shared to the git client. Here is the related documentation: https://docs.gitlab.com/ee/ssh/

**Comment by santiagobasulto** [comment]
@pvojnisek I was able to correctly generate the ssh-key and add it to my Gitlab profile. I just followed the steps in my environment and was able to correctly add it to my gitlab profile:

![image](https://user-images.githubusercontent.com/872296/60760116-69653880-a006-11e9-9439-eadfd33999cb.png)

The problem of course will be that the key will be deleted once the project goes to sleep. In that case, we need to persist the SSH keys for users.

I've created a new issue #21 that supersedes this one. Once #21 is ready, this will be resolved too.
